src/app/cli/src/init/client.ml

```diff
@@ -2169,6 +2169,334 @@ let thread_graph =
                   (humanize_graphql_error ~graphql_endpoint e) ) ;
              exit 1 ) )
 
+let itn_create_accounts =
+  Command.async ~summary:"Fund new accounts for incentivized testnet"
```

Comment:
NIT: maybe move part after parsing parameters to a separate module for not to pollute `client.ml`?